src/api/customer/customer-deserializer.ts

```diff
@@ -9,6 +10,13 @@ const createCustomerDeserializer: CustomerCreationRequestHandler  = (req, res, n
     next();
 }
 
+
+const findCustomersDeserializer: CustomersFindRequestHandler  = (req, res, next) => {
+    res.locals.customerToPK = {uuid: req.params.uuid}
```

Comment:
Poderia ser mais simples:

```suggestion
    res.locals.customerToPK = req.params;
```